### ISSUE TITLE
V1: Unbox and Setup Raspberry Pi 

### ISSUE THREAD

**Issue by nelsonic** [dwyl/video#79]
We have created over 100 tutorials explaining several technical concepts [github.com/search?q=learn](https://github.com/search?q=learn+org%3Adwyl). 
We have never done a tutorial that starts from the very beginning and makes no assumptions about past experience/knowledge. Also, we have not stitched our tutorials together to form a coherent "course" or ["curriculum"](https://github.com/dwyl/the-book/issues/69) that someone can follow at their own pace and build a fully working application. 

> **Note**: Yes, we have tutorials that are aimed at beginners like [github.com/dwyl/phoenix-chat-example](https://github.com/dwyl/phoenix-chat-example) and [github.com/dwyl/javascript-todo-list-tutorial](https://github.com/dwyl/javascript-todo-list-tutorial) but we could not share these with a child (_or anyone who has never written a line of code_) and expect them to _understand_ where to start; they would immediately be overwhelmed. We need to fill in the **`"Hello World"`** gap. 

We need to change this. We need to start from unboxing, plugging in and turning on the computer!
Along the way we need to inform people that a Raspberry Pi is just one of the _many_ options for a computer and that people can follow along with our tutorial series with whichever computer they have.
Our _reasoning_ for using the Raspberry Pi is that it's _really_ affordable ($35), widely available and can be used in a variety of electronics projects.

@iteles we already have the Raspberry Pi 4 Starter. So we have everything we need to do the video.
I would like to record the following:
+ [ ] Unbox the starter kit.
+ [ ] Plug in the power adapter, Micro SD card, keyboard, mouse and  monitor cable.
+ [ ] Turn on the PI
+ [ ] Walk through the Noobs UI with a fresh perspective.
+ [ ] Write a **`"Hello World"`** in whatever programming language is available on the machine.
  We could do `python`, `JavaScript` (`document.write("Hello World")`) or `Scratch`.

I expect the recording of the "talent" parts of the video to take around **1 hour**.
I am happy to provide as little or as much direction in the recording.
Also _very_ happy to follow any ideas that surface for how to make the video _engaging/interesting_.
I Will be the camera person and will do the editing/composition. 
I expect the final video to be around 10 mins long with several sections (e.g boot sequence) sped-up to not bore viewers.

The ultimate objective is a video that ***anyone*** can follow to get their RPi up-and-running.
And then we will encourage people to ask questions if they get stuck. 

Please note: I _know_ there are many videos on YT featuring setting up the RPi. ✅ 
But we need to start at the beginning so that we can make a complete _series_.
Think of this first video as the "back story" where we introduce the main characters.
It might not be the most _interesting_ video (_though we will try our best to spark curiosity_),
but it's absolutely necessary to setup for the future videos.

@iteles please let me know your thoughts, when you have time to discuss/story-board. 📆 
It might be the case that we want to record this in Braga where we have better lights/background ... 💭

**Comment by iteles** [comment]
I think this is a great idea. 
We need to make sure part of the video (a very small, rapid-fire part) gives people some knowledge about what [cool things they can do](https://itsfoss.com/raspberry-pi-projects/) with their [RPi](reporting.xero.com/!MmKfl/filed-online) so they have a reason for getting into this. 
https://projects.raspberrypi.org/en

Although this might not be _critical_ for our _current_ priorities, it's worth investing a specific time block into it because:
a) We have a _lot_ of videos in our future and we need to start making progress on them
b) Most importantly, we need to have a good idea of what is involved and the time required so that in future we can estimate this correctly. Starting with something small and very specific like this is great!
c) It capitalises on our enthusiasm for this idea _now_

@nelsonic Shall we storyboard this towards the end of the week and film over the weekend? 
I agree that Braga will likely have better lighting and audio conditions, so maybe we film next week?

**Comment by nelsonic** [comment]
@iteles agreed. Informing people about the range of projects preferably with images/video borrowed from other channels would be great in the "Why?" section.

The Official Raspberry Pi Channel/Foundation has this setup video: 
[![image](https://user-images.githubusercontent.com/194400/79121423-2f8f4280-7d8d-11ea-8c91-0eb970135e19.png)](https://youtu.be/wjWZhV1v3Pk)
https://youtu.be/wjWZhV1v3Pk
I think we should read the comments and see where people got stuck and address those.

Agree with looking at this toward the _end_ of the week. Just wanted to "capture" it.

**Comment by iteles** [comment]
Great call on the comments section 🚀 🧑‍🔬

**Comment by nelsonic** [comment]
This Newbie Intro to RPi would be improved by removing the random noise in it. 🙄 
[![image](https://user-images.githubusercontent.com/194400/79134623-e8fa1200-7da5-11ea-8cae-badaab4bbb04.png)](https://youtu.be/U7bZWWlqrCo)
https://youtu.be/U7bZWWlqrCo
We can derive some info from it and again some of the comments are relevant.

**Comment by nelsonic** [comment]
[![image](https://user-images.githubusercontent.com/194400/79877882-b0c98380-83e4-11ea-870d-6b50c0fb2508.png)](https://youtu.be/vbaJcRxASo0)
https://youtu.be/vbaJcRxASo0

**Comment by iteles** [comment]
I've found that I've enjoyed having only a super brief history of raspberry pi just to know the evolution (if any history at all) and this 'slide' from the [Newbie Introduction](https://www.youtube.com/watch?v=U7bZWWlqrCo) is pretty quick and concise.

![image](https://user-images.githubusercontent.com/4185328/80114726-ccb35d80-857b-11ea-9dfc-cc820a22e0a3.png)


I've also thought that having a tour of the pi is super useful, although a few go into way too much detail.

**Comment by nelsonic** [comment]
Awesome raspberry pi projects: https://docs.google.com/document/d/1xT6-vN4sFnRLMOdj9wC4_ESWIXbCcSA4Xo0iqJv_j0k
